### DISCUSSION TITLE
Working on Django based project to fine the blood donor.

### DISCUSSION THREAD

**Discussion by GargAnsh** [community/community#69450] [Programming Help]
I am looking for kind support to build my Donor connect project. I am veru new to this. Please suggest ,how can i get the contributer to expend my code.

https://github.com/GargAnsh/donor_finder

**Comment by JEHAD2424** [comment]
Really Mr GitHub a great job very big and new ideas butiful congratulations

**Reply by JEHAD2424** [reply]
Most of the programs as the same way that we used it but you put it in a new way and the questions for the first time very nice

**Reply by JEHAD2424** [reply]
I feel entrust to see all the programs and read all the writing

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @GargAnsh ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information:

>  ❌What is not allowed in this category:
Asking people to code for you

However, as long as you are asking for ways/strategies to get contributors that meets our guidelines.
